tests/e2e/utils/payments.js

```diff
@@ -416,32 +513,24 @@ export const setupACHCheckout = async ( page, checkoutType = 'blocks' ) => {
 	await emptyCart( page );
 	await setupCart( page );
 
+	let iframeSelector = 'iframe[src*="elements-inner-payment"]';
```

Comment:
Nit: rather than overwriting the `iframeSelector` variable, it may be clearer to use a separate variable name for this initial/inner selector, and then use a separate variable for the full iframe selector.
```suggestion
	const rawIframeSelector = 'iframe[src*="elements-inner-payment"]';
```

Reply:
Fixed in [f04f9bc](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4755/commits/f04f9bc41ab3844c32f1af36a2caf7e364415681)